src/main/java/com/conveyal/datatools/manager/jobs/MergeFeedsJob.java

```diff
@@ -325,8 +337,7 @@ private void storeMergedFeed() throws IOException, CheckedAWSException {
                 LOG.info("Storing merged project feed at {}", S3Utils.getDefaultBucketUriForKey(s3Key));
             } else {
                 try {
-                    FeedVersion.feedStore
-                        .newFeed(filename, new FileInputStream(mergedTempFile), null);
+                    FeedVersion.feedStore.newFeed(filename, new FileInputStream(mergedTempFile), null);
                 } catch (IOException e) {
```

Comment:
Deserves a report to bugsnag.

Reply:
Handled in https://github.com/ibi-group/datatools-server/pull/376/commits/8179f873f950f83394f7c628131f9b2e73fc72e1.